src/cli/extensions/helpers.js

```diff
@@ -0,0 +1,281 @@
+import chalk from 'chalk';
+import resolve from 'resolve';
+import { isString } from 'lodash';
+import semver from 'semver';
+
+import { registerHooks, getHooks, setHooks } from '../../hooks';
+import { registerActions, getActions, setActions } from '../../hooks/actions';
+import { merge } from '../../configuration';
+import { warningLabel, feedbackMessage } from '../../helpers/style';
+import ExtensionError from './error';
+
+export function runPostInits(initialState) {
+    return initialState.postInits.reduce(
+        (state, postInit) =>
+            manageRocObject(
```

Comment:
Perhaps a better name  for this is
`registerWithRoc` ?

The function does this:
- Register any hooks, updates the state
- Register any actions, updates the state
- Builds, merges configuration and updates the state

Returns the updated state

Right?

Could also consider breaking these into separate functions. But this works as is, so we can address these things in https://github.com/rocjs/roc/issues/96